### PR TITLE
Update CommitStatus to Status on RepositoriesClient and ObservableRep…

### PR DESCRIPTION
**Pull request by RobPethick**
…ositoriesClient #1034

### CODE REVIEW

File: Octokit/Clients/IRepositoriesClient.cs [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 namespace Octokit
 {
-    /// <summary>
+  using System;
```
Comment (shiftkey) [inline Octokit/Clients/IRepositoriesClient.cs]: :lipstick: move this up outside the `namespace` with the other `using`s

File: Octokit/Clients/IRepositoriesClient.cs [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
+
+  /// <summary>
```
Comment (shiftkey) [inline Octokit/Clients/IRepositoriesClient.cs]: :lipstick: indenting

**Comment by shiftkey** [comment]
A couple of little formatting things, everything else seems :gem:

**Comment by RobPethick** [comment]
Thanks @shiftkey  updated :)

**Comment by Haacked** [comment]
Nicely done! Thanks a lot! :+1:

**Pull request merged**